client/src/js/addendum/AddendumContainer.jsx

```diff
@@ -140,13 +176,47 @@ function AddendumContainer (props) {
           <UserForm callback={userAdded}/>
         </Card>
       </Grid>
-      <Button fullWidth
-              variant='contained'
-              color='primary'
-              disabled={addedIdentities.length === 0 && removedIdentities.length === 0}
-              onClick={createAddendum}>
-        Sign Addendum
-      </Button>
+      <Grid item xs={12}>
+        <Button fullWidth
+                variant='contained'
+                color='primary'
+                disabled={addedIdentities.length === 0 && removedIdentities.length === 0}
+                onClick={createAddendum}>
+          Sign Addendum
+        </Button>
+      </Grid>
+      <Grid item xs={12}>
+        <Dialog
+          onClose={confirmLeave}
+          aria-labelledby="confirm-leave"
+          open={openDialog}>
+          <DialogTitle id="alert-dialog-title">{'Use Google\'s location service?'}</DialogTitle>
+          <DialogContent>
+            <DialogContentText id="alert-dialog-description">
+              You have unsaced changes, are you sure you want to leave?
+            </DialogContentText>
+          </DialogContent>
+          <DialogActions>
+            <Button onClick={() => confirmLeave(false)} color="primary">
+              No, stay and save my changes
```

Comment:
It sounds like changes will be saved by clicking "No"
```suggestion
              No, continue editing
```